src/extension_utils/utils.ts

```diff
@@ -258,37 +258,35 @@ export const checkConfig = (configName: string): boolean => {
 
 export const checkPythonDependencies = async (context: vscode.ExtensionContext, pythonExecutable: string) => {
   let hasInstalledDependencies: boolean = false;
+  const pathToLibs: string = getPathToScript(context, "out", "python_libs");
```

Comment:
It looks like we are using the getPathToScript(context, "out", ...) in a lot of different places. Maybe we can at least define the "out" path somewhere? (E.g. let's say we decide to change the output directory, we would have to change all of its occurrences)

Reply:
I'll make "out" and "python_libs" constants in constants.ts :)